src/screens/FeedbackScreen.tsx

```diff
@@ -0,0 +1,217 @@
+  clicked_emoji_container : ViewStyle,
+  response_container_text : TextStyle,
+  emojis_response_container : ViewStyle,
+  textinput_text : TextStyle,
+  button : ViewStyle,
+  button_text : TextStyle,
+  modal_view : ViewStyle,
+  modal_button : ViewStyle,
+  modal_button_text : TextStyle,
+  modal_text : TextStyle,
+  modal_image : ImageStyle,
+  emoji_text : TextStyle,
+  emoji_image : TextStyle
+}
+
+const styles = StyleSheet.create<FeedbackStyle>({
+    main_container : {
+        flex : 1 ,
+        alignItems : "center",
+        backgroundColor : colors.DROIDCONKE_WHITE
+    },
+    feedback_text : {
+        textAlign : "center",
+        color : colors.DROIDCONKE_BLUE ,
+        width : "80%",
+        margin : 30,
+        fontSize : 18,
+        fontFamily : fonts.MONTSERRAT_BOLD
+    },
+    response_container : {
+        alignItems : "center",
+        marginBottom : 30,
+        borderColor : "#F5F5F5",
+        borderWidth : 1,
+        borderRadius : 8,
+        width : "90%"
+    },
+    emoji_container : {
+        height : 67,
+        width : 67,
+        backgroundColor : colors.DROIDCONKE_PEARL,
+        borderRadius : 4 ,
+        margin : 30 ,
+        justifyContent : "center",
+        alignContent : "center",
+    },
+    clicked_emoji_container : {
+      opacity : 0.2
+    },
+    response_container_text : {
+        color : colors.DROIDCONKE_BLACK,
+        fontSize : 14 ,
+        fontFamily : fonts.MONTSERRAT_REGULAR,
+        textAlign : "center",
+        marginTop : 17,
+    },
+    emojis_response_container : {
+        flexDirection : "row",
+        justifyContent : "space-around"
+    },
+    textinput_text : {
+        backgroundColor : colors.DROIDCONKE_PEARL,
+        width : "90%",
+        marginBottom : 30,
+        height : 115,
+        textAlign : "left",
+        textAlignVertical : "top",
+        padding : 5
+    },
+    button : {
+        backgroundColor : colors.DROIDCONKE_BLUE,
+        alignItems : "center",
+        justifyContent : "center",
+        width : "90%",
+        borderRadius : 10,
+        height : 45
+    } ,
+    button_text : {
+        fontSize : 16,
+        fontFamily : fonts.MONTSERRAT_SEMIBOLD,
+        color : colors.DROIDCONKE_WHITE,
+        textAlign : "center"
+    },
+    modal_view: {
+        backgroundColor: colors.DROIDCONKE_WHITE,
+        borderRadius: 20,
+        padding: 35,
+        alignItems: "center",
+        elevation: 5,
+        justifyContent: "center",
+        alignSelf : "center",
+        marginTop : Dimensions.get("screen").height / 5.5
+    },
+    modal_button: {
+        justifyContent : "center",
+        alignContent : "center",
+        width : 160,
+        backgroundColor : colors.DROIDCONKE_BLUE,
+        height : 45,
+        borderRadius : 10,
+        marginTop : 25
+    },
+    modal_button_text: {
+        color: colors.DROIDCONKE_WHITE,
+        fontWeight: "bold",
+        textAlign: "center"
+    },
```

Comment:
the modal_button_text also has
`fontSize : 16,
fontFamily : MONTSERRAT_SEMIBOLD,`

PS: It actually uses the same styles as button_text 🤭​.